src/Element/Note.css

```diff
@@ -1,43 +1,55 @@
 .note.thread {
-    border-bottom: none;
+  border-bottom: none;
 }
 
-.note > .header .reply {
-    font-size: var(--font-size-tiny);
-    color: var(--font-secondary-color);
+.note>.header .reply {
+  font-size: var(--font-size-tiny);
+  color: var(--font-secondary-color);
 }
 
-.note > .header > .info {
-    font-size: var(--font-size);
-    white-space: nowrap;
-    color: var(--font-secondary-color);
+.note>.header>.info {
+  font-size: var(--font-size);
+  white-space: nowrap;
+  color: var(--font-secondary-color);
 }
 
-.note > .body {
-    margin-top: 12px;
-    padding-left: 56px;
-    text-overflow: ellipsis;
-    white-space: pre-wrap;
-    word-break: normal;
-    overflow-x: hidden;
-    overflow-y: visible;
+.note>.body {
+  margin-top: 12px;
+  padding-left: 56px;
+  text-overflow: ellipsis;
+  white-space: pre-wrap;
+  word-break: normal;
+  overflow-x: hidden;
+  overflow-y: visible;
 }
 
-.note > .footer {
-    padding-left: 46px;
+.note>.footer {
+  padding-left: 46px;
 }
 
-.note > .header img:hover, .note > .header .name > .reply:hover, .note .body:hover {
-    cursor: pointer;
+.note>.footer .ctx-menu {
+  background-color: var(--note-bg);
+  color: var(--font-secondary-color);
+  border: 1px solid var(--font-secondary-color);
+  border-radius: 16px;
+}
+
+.note>.header img:hover, .note>.header .name>.reply:hover, .note .body:hover {
+  cursor: pointer;
 }
 
-.note > .note-creator {
+.note>.note-creator {
   margin-top: 12px;
```

Comment:
I think we should pad this so is aligned with note body.

```suggestion
  margin-top: 12px;
  padding-left: 56px;
```

Reply:
![image](https://user-images.githubusercontent.com/1172179/213865857-14c88c97-dc60-409e-b06a-ea77fbbd6a93.png)